rules.md

```diff
@@ -124,7 +124,7 @@ The core of the system is the concept of a Bounty which is a grouping of Tasks t
 
 Every piece of output from the Incubator is created via a Bounty, whether that’s a Dapp that’s produced, a website that’s hosted, or a Funding Proposal raised to the Dash Network, it’s tracked as a Bounty managed by Admins in the App.
 
-Each Bounty is represented by an individual Card in the App and refers to a piece of work proposed as a Concept by an App user.  Bounties are categorized into one of the three types mentioned in the Overview: Project, Service or Job.  Each Bounty comprises a set of Tasks grouped into one of the four Task Types: Concept, Specification, Production and QA Tasks, described later.  Every Bounty has a Bounty Admin, which is the Admin user incentivized to create and manage the Bounty’s Tasks for a commission added as a % on all Task rewards.
+Each Bounty is represented by an individual Card in the App and refers to a piece of work proposed as a Concept by an App user.  Bounties are categorized into one of the three types mentioned in the Overview: Project, Service or Job.  Each Bounty comprises a set of Tasks grouped into one of the four Task Types: Concept, Specification, Production and QA Tasks, described later.  Every Bounty has two Bounty Admins, a Primary Admin and a Secondary Admin. [Admins](#4-admins) are users incentivized to create and manage the Bounty’s Tasks.  They earn [commission](#44-admin-rewards) on all completed Bounty Tasks.
```

Comment:
Ambiguity? Does this imply that both admins each earn the specified rewards when tasks are completed?

Reply:
I didn't want to add detail to this overview section, which is why I linked to the Admin and commission sections.  This could potentially be resolved by restructuring the document, but that's out of scope for this PR/change.  If we want to be explicit here (even if that means being redundant) we could replace the following:
> They earn [commission](#44-admin-rewards) on all completed Bounty Tasks.

with something like:
> An admin earns [commission](#44-admin-rewards) by approving Bounty Tasks, which for a given task is carried out by either the Primary Admin or the Secondary Admin (but not both; only one admin earns the commission for a given task).
Thoughts?

Reply:
Yes, I agree, this is probably a matter for a future PR addressing some of the existing grammatical & other idiosyncrasies, rather than the subsatnce of your changes. We can refer back to it when addressing those at a later date.